frontend/test/metabase/visualizations/components/LineAreaBarRenderer.unit.spec.js

```diff
@@ -166,7 +124,12 @@ describe("LineAreaBarRenderer", () => {
 
     const cols = [dateColumn, NumberColumn()];
     const chartType = "line";
-    const series = [{ data: { cols, rows }, card: { display: chartType } }];
+    const series = [
+      {
+        data: { cols, rows },
+        card: { display: chartType, report_timezone: "Etc/UTC" },
```

Comment:
`report_timezone` was removed right?

Reply:
Good catch! It was working anyways because "Etc/UTC" is the default anyways. I just deleted it here.